### ISSUE TITLE
Description of Standard for Public Code includes a lot of stewardship explanation

### ISSUE THREAD

**Issue by clausmullie** [publiccodenet/publiccode.net#322]
The repository for the Standard for Public Code is made as neutral as possible - it contains only the standard, with all the stewardship related things moved into About. I remember this was to make the Standard a stand alone asset anyone can use and reuse as they see fit, while stewardship is something particular to our organization.

However, I get the sense this separation is not made as clearly on the homepage. The [introduction to the standard page](https://publiccode.net/standard-for-public-code/) includes a lot of "how do we do stewardship" information that perhaps lives best on the [introduction to stewardship page](https://publiccode.net/codebase-stewardship/).

**Comment by Ainali** [comment]
> The [introduction to the standard page](https://publiccode.net/standard-for-public-code/) includes a lot of "how do we do stewardship" information that perhaps lives best on the [introduction to stewardship page](https://publiccode.net/codebase-stewardship/).

When you say "introduction", do you mean the text before the "Standard compliance or certification process" section?

**Comment by clausmullie** [comment]
> When you say "introduction", do you mean the text before the "Standard compliance or certification process" section?

No, I meant the entire page. But on second thought, I'm doubting my original issue.

**Comment by Ainali** [comment]
I realize, though, that there isn't much of a text overview/introduction. The video has it all, but for someone just reading, it goes from the lead straight into the certification. One more paragraph under the video, summarizing it, would be a great addition.

**Comment by Ainali** [comment]
> * I noticed this currently doesn't link to the standard, might be good to add that at the bottom

It links to standard.publiccode.net on line 37, but perhaps that link should go to the page about the standard on publiccode.net instead?
